spec/models/user_spec.rb

```diff
@@ -302,6 +159,19 @@
         to raise_error(ActiveRecord::RecordInvalid,
                        /Validation failed: Password is too long \(maximum is 128 characters\)/)
     end
+
+    it 'works with spaces' do
+      PASSWORD_W_SPACES = 'this has a few spaces'.freeze
+
+      prototype_user = create(:user)
+
+      # Verify success.
+      user = create(:user,
+                    email: "mkuniq.#{prototype_user.email}",
+                    password: PASSWORD_W_SPACES,
+                    password_confirmation: PASSWORD_W_SPACES)
+      expect(user.errors.any?).to be_falsey
```

Comment:
One more thing. You don't need to hit the database for this test. You can use the faster `build_stubbed` method.

```
user = build_stubbed(:user, ....)
```